cost-model/src/cost_model.rs

```diff
@@ -151,7 +151,7 @@ impl CostModel {
 
     fn get_transaction_cost(
         tx_cost: &mut UsageCostDetails,
-        transaction: &SanitizedTransaction,
+        transaction: &impl SVMTransaction,
```

Comment:
small nit - I think we technically only need an `SVMMessage` here. We don't use the additional signature accessors from `SVMTransaction` afaict

Reply:
5640a49ba1b6a0c39c3133b19abc953e6fbdadd6